product_docs/docs/pgd/5/cli/command_ref/cluster/show.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: pgd cluster show
+navTitle: Show
+---
+
+## Synopsis
+
+The `pgd cluster show` command is used to display the cluster-level information in the EDB Postgres Distributed cluster.
+
+## Syntax
+
+```plaintext
+pgd cluster show [OPTIONS]
+```
+
+## Options
+
+The following table lists the options available for the `pgd events show` command:
+
+| Short       | Long                | Description                                             |
+|-------------|---------------------|---------------------------------------------------------|
+|             | --clock-drift       | Only show detailed clock drift information.             |
+|             | --summary           | Only show cluster summary information.                  |
+|             | --health            | Only show cluster health information.                   |
+
+Only one of the above options can be specified at a time.
+
+See also [Global Options](/pgd/latest/cli/command_ref/#global-options).
+
+## Examples
+
+### Display the cluster information
+
+```shell
+pgd cluster show
+__OUTPUT__
+# Summary
+Group Name Parent Group Group Type      Node Name  Node Kind
+---------- ------------ --------------- ---------- ---------------
+pgd                     global
+dc1        pgd          data            node-one   data
+dc1        pgd          data            node-three data
+dc1        pgd          data            node-two   data
+sogroup1   pgd          subscriber-only sonode-one subscriber-only
+sogroup1   pgd          subscriber-only sonode2    subscriber-only
+
+# Health
+Check             Status Message
+----------------- ------ -----------------------------------------------
+Connections       Ok     All BDR nodes are accessible
+Raft              Ok     Raft Consensus is working correctly
+Replication Slots Ok     All PGD replication slots are working correctly
+Clock Skew        Ok     Clock drift is within permissible limit
+Versions          Ok     All nodes are running the same PGD version
+
+# Clock Drift
+Reference Node Node Name  Clock Drift
+-------------- ---------- -----------
+node-one       node-two   *
```

Comment:
We should document the meanin g of the special symbols displayed as `Clock Drift` value similar to legacy CLI